index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, intial-scale=1.0"/>
+    <title>First Portfolio</title>
+    <link href="./style.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+    <!-- START NAVBAR SECTION-->
+    <nav>
+        <div class="navbar">
+            <img src="./img/image_geometry_1.svg" alt="image1"/>
```

Comment:
Kindly, I recommend you to put all the decorative images in the background of your website, with the "background-image" property, it can be only 1 image and adapt it to the size of the device or put several background images and position them in their corresponding places.